packages/sdks/e2e/angular/src/app/app.component.ts

```diff
@@ -78,7 +74,5 @@ export class AppComponent {
     this.apiKey = builderProps.apiKey;
     this.model = builderProps.model;
     this.apiVersion = builderProps.apiVersion;
-
-    this.cdr.detectChanges();
```

Comment:
![CleanShot 2024-08-01 at 12 33 42@2x](https://github.com/user-attachments/assets/43fccbc6-2881-41b1-8781-cdf7c0766daa)

i dont know if this passed by luck!

if it's flakey, might have to use `test.skip()` instead of `test.fail()`

Reply:
yes we should add `test.skip()` to it I think, because A/B tests are not working yet